content/_partials/carousel.html.haml

```diff
@@ -21,6 +21,14 @@
   })
 
 %div#ji-home-carousel.carousel.slide{'data-ride' => 'carousel',:class => page.topic}
+#ji-download
```

Comment:
@arshdkhn1 I didn't catch this in the original review, the site compilation failure you posted to the `jenkinsci-dev@` mailing list is due to the indentation of this Haml.

Haml uses indentation to know how to nest tags when it renders, so the `.btn-group` here is too far indented under `#ji-download`.

I'm fixing this in a follow-up pull request right now